src/index.html

```diff
@@ -13,10 +13,18 @@
     <title>Stop watch</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/styles.scss"
```

Comment:
The `href` attribute in the `<link>` tag should point to a CSS file, not a SCSS file. Please ensure that the path is correct and points to a compiled CSS file.